src/NuGet.Clients/NuGet.VisualStudio.Common/NuGet.VisualStudio.Common.csproj

```diff
@@ -31,7 +31,8 @@
     <PackageReference Include="Microsoft.VisualStudio.Services.InteractiveClient" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.15.0" />
     <PackageReference Include="Microsoft.VisualStudio.Shell.Immutable.15.0" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.16.0.DesignTime" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.16.0.DesignTime"/>
```

Comment:
You are missing a space at the end of the xml line.

Reply:
Updated.